controllers/registrations.js

```diff
@@ -128,4 +128,28 @@ registrationsRouter.get('/current', checkAdmin, (req, res) => {
     .catch((error) => handleDatabaseError(res, error))
 })
 
+registrationsRouter.get('/', checkLogin, (req, res) => {
+  const token = getTokenFrom(req)
+  const decodedToken = jwt.verify(token, config.secret)
```

Comment:
From `jsonwebtoken`'s [docs](https://github.com/auth0/node-jsonwebtoken/tree/da8f55c3c7b4dd0bfc07a2df228500fdd050242a#jwtverifytoken-secretorpublickey-options-callback):

> (Synchronous) If a callback is not supplied, function acts synchronously. Returns the payload decoded if the signature is valid and optional expiration, audience, or issuer are valid. **If not, it will throw the error.**

Could add a try..catch?

Reply:
nvm apparently checkLogin already does this